src/app/store.ts

```diff
@@ -1,6 +1,13 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The import statement `combineSlices` seems incorrect. It should be `combineReducers` from `@reduxjs/toolkit` to properly combine the reducers.